docs/src/architecture.md

```diff
@@ -1,52 +1,50 @@
 
-We believe that to attract billions of users to Ethereum, we need to extend it on three dimensions: scalability, safety, and privacy. 
-
-The actor model is our inspiration for achieving concurrent, local state changes on a distributed system like a blockchain. In the actor model, actors play the role of little state machines, meaning each actor is responsible for their own state. Actors have inboxes to send and receive messages to communicate with other actors. Messages can be read asynchronously.
+The [Actor Model](https://en.wikipedia.org/wiki/Actor_model) is our inspiration for achieving concurrent, local state changes on a distributed system like a blockchain. In the actor model, actors play the role of little state machines, meaning each actor is responsible for their own state. Actors have inboxes to send and receive messages to communicate with other actors. Messages can be read asynchronously.
 
 ## Concepts in Miden
-Users can interact on the network executing transactions. Sucessfull transactions trigger a state change. Transactions are always executed against single accounts and produce or consume at least one note. This is similar to the [Actor Model](https://en.wikipedia.org/wiki/Actor_model). The state of all accounts and notes is committed to via the L2 state root which is being published and accompanied by a ZK proof and additional data to Ethereum. This is similar to any other zkRollup. 
+Users can interact on the network executing transactions. In Polygon Miden, there are accounts and notes, both of which can hold assets. Transactions can be understood as facilitating account state changes. Basically, they take one single account and some notes as input and output the same account at a new state together with some other notes. This is quite different from what we see on Ethereum.
 
 The concepts which constitutes the Miden Architecture are 
 
-* Accounts and Notes and the Transaction Model
-* State and Execution Model
+* Accounts and notes and the transaction model
+* State and execution model
 
 ## Transaction life cycle 
-To illustrate the core protocol, let's look at how Alice can send Bob 5 MATIC in Polygon Miden. The basic building blocks of Polygon Miden are Accounts, Notes and Transactions. 
+To illustrate the core protocol, let's look at how Alice can send Bob 5 MATIC in Polygon Miden. The core concepts of Polygon Miden are Accounts, Notes and Transactions. 
 
 _Note: Because of the asynchronous execution model two transactions are needed to transfer the assets._
 
 <p align="center">
-    <img src="./diagrams/architecture/transaction_lifecycle/Account_Alice_1.svg">
+    <img src="./diagrams/architecture/transaction_lifecycle/Account_Alice_1.png">
```

Comment:
Would this (and other) pictures show up correctly? Seems like we use white color to draw - but that's the same as the default background color for docs.

Reply:
I think you are right. I changed it back. We should involve a designer soon. Like this is looks slightly unprofessional